src/main/java/com/eleks/auctioneer/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.eleks.auctioneer.controller;
+
+import com.eleks.auctioneer.DTO.AppUserDTO;
+import com.eleks.auctioneer.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @PostMapping("/posts")
+    public String addNewUser(@Validated @RequestBody AppUserDTO userInfoDTO) {
```

Comment:
It's a bad practice to return string. If you want to return some message, create a DTO. E.g. ResponseStatus or something like that and put a single field "message" here.